frame/stake-tracker/README.md

```diff
@@ -1,6 +1,9 @@
 # Pallet `stake-tracker`
```

Comment:
```suggestion
# Pallet `staking-stake-tracker`
```

Just `state-tracker` sounds very generic - like it could track any state.

Reply:
The confusing part is, it sounds like `state`-tracker but its `STAKE`-tracker :D

Reply:
I have made that typo several times myself. Maybe `pallet-staking-tracker` but that doesn't sound as accurate since we literally only keep track of the stake of nominators and validators.